include/ck/ck.hpp

```diff
@@ -159,6 +159,11 @@
 // tuning parameter
 #define CK_WORKAROUND_SWDEV_325164 0
 
+// workaround: disable broken fused attention kernel instance that does not pass validation
+// issue found on mi100/#10738 combo when irregular KPerBlock attention kernel has acc0 scaling
+// enabled
+#define CK_WORKAROUND_DISABLE_BROKEN_ATTN_KERNEL_INSTANCE 1
```

Comment:
Is it still broken on rocm5.3?
Is there a compiler JIRA for it?

@rosenrodt

Reply:
ROCm 5.3 on MI100 looks good. I'll remove it in next PR.